src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+    private static User user1;
+
+    @BeforeAll
+    public static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        user1 = new User("spider@gmail.com", "qwerty12345", 25);
+        registrationService.register(new User("spider@gmail.com", "465s4df6", 35));
+    }
+
+    @Test
+    public void register_AlreadyExistedUser_notOk() {
```

Comment:
```suggestion
    public void register_alreadyExistedUser_notOk() {
```
Let's better add the same user inside this method, and use for this purpose people.add method